### DISCUSSION TITLE
Allow panning of map

### DISCUSSION THREAD

**Discussion by davidpcahill** [voxeltycoon/issues#1037] [General]
Please allow panning of the map over all areas. Having the camera jerk to navigate around is jarring and doesn't add anything to game play. If you really want to hide stuff (though not sure why) I'd suggest you use fog of war (could do it with pretty voxel clouds even). 

Mini side suggestion if you are already in the camera controller code anyways: Also please remove the limit on the camera tilt (and zoom) so we can view things from any angle.

**Comment by IDragonfire** [comment]
Did you try the mod Unrestricted Camera?
